### ISSUE TITLE
Added key went away

### ISSUE THREAD

**Issue by drt24** [ucam-cl-dtg/urop-2014-ticking-ui#57]
I added a key "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDHt9wYQcmbir0YXnlcCFco+NRBujO2lAd47jQevej2WhpxFKXfaqL0ffBlj/1UGkVdeGXa3tAiCDIFFHdcYJtj48Cy2YVm1KOyYtlS1LMdenqpYjZgyE6JT0UCGM+6sfQCcrqkgebkyDL1mpOrnY09zhHA0PQYeqLZgC5BOIQQsMSkX1ZuYh0XJEPlScGBET9nt5yvOubDazwb6/If06vyCmjF4BN/YQgkSwwUQgG2oTBb/EUVFUzkxSYXo2XNMVLMTO94TuvksB2/SdDPffYUTPPQ/iBNASy2agZbbV1PGM24Bu6zzY8ut5cpUUBoSlje4jkt/dnQvx3AwQ4cc7rbQgkHEUVeH+hj169qW4Xp4IZkcfGDZnM0lgu9/hBk913GuVObR/qI+S+Mp2nURM4OGDRIA+HvWXx/J4wxMUkQvCMJyijzlBkhKqFSOqrBleZJtVWyw23zRcalNtu7F9yloodox4crJBxd2ysrnIlntc1upiib3pG6QbOPWauk82/wSs1iRq1de28O0lzebB4QhFd5OAYkQhazpFTI2mXatdkgz0at/KuzPEZjeDAf9rpYm70KkI3MMe5n4USgFftXeo5fspo2AtjNSsonTm9YtI3TFDi4+AQLugEHUuvfbeH0CsCWSHemzbY1RaDyoQY+9u7Wl8lOq1y+GKG/EYSC0w==" and pressed the green button and got a 'Successfully added key' toast but then when I refreshed the page it went away.

**Comment by TomLefley** [comment]
You key is in the database as the above, so I'm not sure why it's not appearing for you, I'll look into it though

**Comment by drt24** [comment]
It is now added, it worked the second time around.

**Comment by TomLefley** [comment]
This was very likely because of a server deploy that could have been happening at the same time as I've not been able to reproduce the issue, I will keep this open until we are sure though.

**Comment by TomLefley** [comment]
Yeah I can't reproduce this, I'm assuming it was the above, I will move this to post-friday unless it comes up again
